userguide/plugins.rst

```diff
@@ -96,9 +96,12 @@ after a successful install. Installed plugins appear in the
 :menuselection:`Plugins --> Installed`
 page as shown in :numref:`Figure %s <view_installed_plugins_fig>`.
 
-.. tip:: Installed plugins are also added to the
+.. note:: Plugins are also added to
    :menuselection:`Jails`
-   page. This page is also used to manage installed software.
+   as a *pluginv2* jail. This jail type is adjustable just like a
```

Comment:
The jail type is adjustable?  I think this means "This type of jail is editable like a"
```suggestion
   as a *pluginv2* jail. This type of jail is editable like a
```